catalyst/utils/hydra_sys.py

```diff
@@ -0,0 +1,64 @@
+import json
+import logging
+from pathlib import Path
+
+from omegaconf import DictConfig, OmegaConf
+
+from catalyst.contrib.tools.tensorboard import SummaryWriter
+from catalyst.utils.config import save_config
+from catalyst.utils.sys import (
+    get_environment_vars,
+    list_conda_packages,
+    list_pip_packages,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def dump_environment(cfg: DictConfig, logdir: str) -> None:
```

Comment:
how we could merge it with the current `dump_environment` implementation?
is it possible to just add extra hydra mixins with `IS_HYDRA_AVAILABLE` flag?

Reply:
I'll try.